runner.py

```diff
@@ -283,6 +283,9 @@ def runner(input_dir, reference_file, output_dir, max_basecall_iterations, min_c
         db_path = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'db')
     if not output_dir:
         output_dir = assign_output_dir(db_path)
+    if os.path.exists(output_dir): # the output folder must be empty
+        if os.listdir(output_dir):
+            raise Exception("The output folder is not empty. Empty it or choose a different name.")
```

Comment:
please maintain consistency in variable names - "output folder" should be "output_dir".
text is also not clear enough.
Recommend changing to something like: "output_dir must be path to a new or empty directory!"